tests/reporter/kubernetes.go

```diff
@@ -295,7 +295,7 @@ func (r *KubernetesReporter) logDMESG(virtCli kubecli.KubevirtClient, logsdir st
 			commands := []string{
 				virt_chroot.GetChrootBinaryPath(),
 				"--mount",
-				virt_chroot.GetChrootMountNamespace(),
+				virt_chroot.GetChrootBinaryPath(),
```

Comment:
Can you explain this change?

Reply:
I think I made a mistake here, responding to @vasiliy-ul comments above saying it better to reference the `/proc/1/ns/mnt` for the test suite. I should have used `mountNamespace` not the `binarypath`. Will modify.

Reply:
updated the patch.